pilot/helpers/agents/Developer.py

```diff
@@ -67,7 +67,7 @@ def start_coding(self, task_source):
             num_of_tasks = len(self.project.development_plan)
             # don't create documentation for features
             if not self.project.finished:
-                current_progress_percent = round(((i + 1) / num_of_tasks) * 100, 2)
+                current_progress_percent = round(((i) / num_of_tasks) * 100, 2)
```

Comment:
Isn't `i` still zero-based? Also, if this is now correct, no need to put `i` in parentheses.